### ISSUE TITLE
`Data.Traversable` re-exports `sum` from `Data.Foldable` but not `product`

### ISSUE THREAD

**Issue by mhmdanas** [purescript/purescript-foldable-traversable#137]
I find this to be a bit strange. Is there any difference that warrants one being re-exported but not the other, or was this an oversight?

**Comment by hdgarrood** [comment]
Yeah, it’s definitely weird to re-export sum but not product. I’d suggest we start re-exporting product again (the Foldable one), and using qualified imports for the functors one.

**Comment by JordanMartinez** [comment]
I'll label this as breaking

**Comment by hdgarrood** [comment]
I think this is only breaking if we are currently re-exporting a `product` with a different type, but it sounds like we are not currently doing that?

**Comment by mhmdanas** [comment]
I guess it might also be breaking if somebody has an open import of `Data.Traversable` and they also import into scope a different `product`.

**Comment by hdgarrood** [comment]
We don’t consider that breaking because the compiler will give you warnings if your imports are set up in such a way that a name being added in a module you’re importing could cause a failure.
